src/dispatchers/mouseDispatcher.ts

```diff
@@ -2,13 +2,19 @@
 
 module Plottable {
 export module Dispatcher {
+  export type MouseCallback = (p: Point) => any;
+
   export class Mouse extends AbstractDispatcher {
     private static _DISPATCHER_KEY = "__Plottable_Dispatcher_Mouse";
     private _svg: SVGElement;
     private _measureRect: SVGElement;
     private _lastMousePosition: Point;
     private _moveBroadcaster: Core.Broadcaster<Dispatcher.Mouse>;
-    private _processMoveCallback = (e: MouseEvent) => this._processMoveEvent(e);
+    private _downBroadcaster: Core.Broadcaster<Dispatcher.Mouse>;
+    private _upBroadcaster: Core.Broadcaster<Dispatcher.Mouse>;
+    private _processMoveCallback: (e: MouseEvent) => any;
+    private _processDownCallback: (e: MouseEvent) => any;
+    private _processUpCallback: (e: MouseEvent) => any;
```

Comment:
Missed this comment. I think they return `void`? Although `AbstractDispatcher` only specifies that they should return `any.